crates/sui-graphql-rpc/src/types/transaction_block_kind/consensus_commit_prologue.rs

```diff
@@ -22,10 +24,17 @@ use sui_types::{
 pub(crate) struct ConsensusCommitPrologueTransaction {
     epoch: u64,
     round: u64,
+    /// The sub DAG index of the consensus commit. This field will be populated if there
+    /// are multiple consensus commits per round.
+    sub_dag_index: Option<u64>,
```

Comment:
I think adding this field in the middle of this struct should be ok, since we never serialize it.

Reply:
cc @amnn

Reply:
Hi @amnn , I'm going to merge this PR for now since I'm working on another PR that is based on this one. But would really appreciate it if you can take a look at the graphql related places again, especially now we add another field in consensus commit prologue V3. Thanks!